test/xMsExamplesExtractor/databox.json

```diff
@@ -0,0 +1,2157 @@
+{
```

Comment:
should the spec be simplified to target what we need to test? seems like a long spec.

Reply:
should we have a folder "databox"  or "simple" and put all the assets corresponding to it there? the swagger and session records inside?

Reply:
Since we had no tests for x-ms-examples-extractor, I would prefer to keep at least one set of files that gives us good coverage.